src/skagent/bellman.py

```diff
@@ -71,6 +71,75 @@ def decision_function(
         post = self.block.transition(vals, decision_rules)
         return {sym: post[sym] for sym in decision_rules}
 
+    def _compute_gradients_for_tensors(self, tensors_dict, wrt):
```

Comment:
`self` is not used in this method body. would this be better a standalone function, perhaps in `utils.py`?